guides/v2.2/cloud/project/project-multi-sites.md

```diff
@@ -8,157 +8,246 @@ functional_areas:
-    {
-        $_SERVER["MAGE_RUN_CODE"] = "french";
-        $_SERVER["MAGE_RUN_TYPE"] = "website";
-    }
-    ```
-
-1.  Save your changes to the `magento-vars.php` file.
+       }
+       return strpos(str_replace('---', '.', $_SERVER['HTTP_HOST']), $host) === 0;
+   }
+   ```
+   {:.no-copy}
+
+1. Change the following line:
+
+   From:
+
+   ```php?start_inline=1
+   return strpos(str_replace('---', '.', $_SERVER['HTTP_HOST']), $host) === 0;
+   ```
+
+   To:
+
+   ```php?start_inline=1
+   return $_SERVER['HTTP_HOST'] ===  $host;
+   ```
+
+1. Replace the following values in the `if (isHttpHost("example.com"))` block:
+   -  `example.com`—with the base URL of your website
+   -  `default`—with the unique code for your website or store view
+   -  `store`—with one of the following values:
+      -  `website`—load the website in the storefront
+      -  `store`—load a storeview in the storefront
+
+   ```php
+   <?php
+   function isHttpHost($host)
+   {
+       if (!isset($_SERVER['HTTP_HOST'])) {
+       return false;
+       }
+       return $_SERVER['HTTP_HOST'] ===  $host;
+   }
+   if (isHttpHost("second.store.com"))
+   {
+       $_SERVER["MAGE_RUN_CODE"] = "<second-site>";
+       $_SERVER["MAGE_RUN_TYPE"] = "website";
+   }elseif (isHttpHost("store.com")){
+       $_SERVER["MAGE_RUN_CODE"] = "base";
+       $_SERVER["MAGE_RUN_TYPE"] = "website";
+   }
+   ```
+
+   For multiple sites with the same domain:
+
+   ```php
+   <?php
+   function isHttpHost($host)
+   {
+       if (!isset($_SERVER['HTTP_HOST'])) {
+       return false;
+       }
+       return $_SERVER['HTTP_HOST'] ===  $host;
+   }
+   if (isHttpHost("store.com")) {
+      $code = "base";
+      $type = "website";
+
+      $uri = explode('/', $_SERVER['REQUEST_URI']);
+      if (isset($uri[1]) && $uri[1] == 'second') {
+        $code = '<second-site>';
+        $type = 'website';
+      }
+      $_SERVER["MAGE_RUN_CODE"] = $code;
+      $_SERVER["MAGE_RUN_TYPE"] = $type;
+   }
+   ```
+
+1. Save your changes to the `magento-vars.php` file.
 
 ### Deploy and test on the Integration server
 
-The final step is to push your changes to your {{site.data.var.ece}} server and test your site there.
+Push your changes to your {{site.data.var.ece}} Integration environment and test your site.
+
+1. Add, commit, and push code changes to the remote branch.
 
-1.  Add, commit, and push code changes to the remote branch.
+   ```bash
+   git add -A && git commit -m "Implement multiple sites" && git push origin <branch-name>
+   ```
 
-    ```bash
-    git add -A && git commit -m "Implement multiple sites" && git push origin <branch name>
-    ```
+1. Wait for deployment to complete.
 
-1.  Wait for deployment to complete.
+1. After deployment, open your site base URL in a web browser.
 
-1.  After deployment, open your site base URL in a web browser.
+   Use the format: `http://<magento-run-code>.<site-URL>`
```

Comment:
is this section only for stores with subdomains? if no, probably need to rephrase like "open store  URL based on configuration you made, for example for subdomains use format..."